src/scales/scale.linear.js

```diff
@@ -12,6 +12,31 @@ module.exports = function(Chart) {
 	};
 
 	var LinearScale = Chart.LinearScaleBase.extend({
+
+		scaleLimitSnapshot: {
+			max: null,
+			min: null
+		},
+
+		/**
+		 * A util function to create snapshot of axis limits {max, min}
+		 * @Method chart.takeScaleLimitSnapshot
+		 * @param Integer {min} chart mininmum
+		 * @param Integer {max} chart maximum
+		 * @return Nothing - no return value
+		 * */
+		takeScaleLimitSnapShot: function(min, max) {
```

Comment:
Agree, outside the scale prototype: `function takeScaleLimitSnapShot(min, max) { ... ` Better API and minification!